runtime/opt/taupage/init.sh

```diff
@@ -94,5 +94,22 @@ else
     echo "ERROR: $RUNTIME failed to start with exit code $result ($ELAPSED_SECONDS seconds elapsed)"
 fi
 
-# finished!
-exit $result
+# FIXME: should be improved by checking for on-exit and on-success
+if [ "$config_shutdown" ] && [ "$config_shutdown" != "never" ]; then
+  echo "INFO: waiting for taupage"
+  EXITCODE=$(docker wait taupage)
```

Comment:
you're right... will fix this. Also found out that it's important to do the test with on-success. Cause `docker wait` exits with 1 of the name is unknown.